### PR TITLE
To do list project 

### PR DESCRIPTION
**Pull request by rahalrazika**
A To-do list is an app build with js :

- Built with Webpack
- Created all modules
- for more details please see the readme file.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello Team 👋

**You've both done well here. Well structured and modular code 👌**

However, let's make some changes.

- [x] Currently, it isn't possible to create a new project clicking the `create project` button after filling all form fields returns no action. Kindly check and fix.

- Please refer to other comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+});
+
+//
+
+// document.querySelector('#content').innerHTML = ListProjects();
+
+// let project = JSON.parse(window.localStorage.getItem('projects'));
+
+
+// document.querySelector('#content').innerHTML = NewItem(project[0]);
```
Comment (mikenath223) [inline src/index.js]: - [x] Please remove these commented code lines. If you want to refer to previous versions of the code consider using git history.

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED ♻️ 🛑 

Hello, @rahalrazika and @lirad very nice work on this project 👍🏿 . I love the way you have separated your code into modules. 👍🏿  👏🏿 

- [x] When I try to create a to-do, your application throws an error. ⬇️ 
     
Screenshot from 2021-01-15 12-12-46

The error seems to be in your local storage file. The save project to function. ⬇️  Since as you can see below the local storage is not created. Kindly fix this.

Screenshot from 2021-01-15 12-13-23

Nb: because of the above error your application is not working hence I was not able to review other parts of the application.

- [x] In your readme file kindly add how to run your project locally.

### if you have any question, kindly reach me on slack @peterrobert

### HAPPY CODING 💻

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED ♻️  🛑 

Hello again, Nice work working on the previous error 👍🏿  Your project looks amazing. Just a few more changes and your project will be good to go. Kindly check below ⬇️ 

- [x] There should be a default project. At the moment, when a person views your application for the first time, there is a notice telling us that we don't have any project. However, this should not be the case. Kindly create a default project 😄 

### [OPTIONAL]

- [x]  In your package.json, kindly add some more common commands like:  ` "start": "webpack-dev-server --mode development --open" `


### If you have any question kindly reach me on slack @peterrobert 

### HAPPY CODING 💻

**Review by addod19 — APPROVED** [review]
## Approved :trophy: :balloon:

Hi @rahalrazika and @lirad, congratulations on an amazing job done :+1: Having fix all the comments made by the previous TSE's and having met all the minimum requirements :slightly_smiling_face: I hereby approve the project :100: 

Go ahead and merge into the development :+1: 
Do not forget to submit the project completion form
_Happy coding_ :woman_technologist: :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**